user-service/src/main/java/rs/edu/raf/banka1/mapper/CompanyMapper.java

```diff
@@ -16,7 +17,20 @@ public CompanyDto toDto(Company company) {
         companyDto.setIdNumber(company.getIdNumber());
         companyDto.setJobId(company.getJobId());
         companyDto.setRegistrationNumber(company.getRegistrationNumber());
-
+        companyDto.setAddress(company.getAdress());
         return companyDto;
     }
+
+    public Company createCompany(CreateCompanyDto createCompanyDto) {
+        Company company = new Company();
+        company.setCompanyName(createCompanyDto.getCompanyName());
+        company.setTelephoneNumber(createCompanyDto.getTelephoneNumber());
+        company.setFaxNumber(createCompanyDto.getFaxNumber());
+        company.setPib(createCompanyDto.getPib());
+        company.setIdNumber(createCompanyDto.getIdNumber());
+        company.setJobId(createCompanyDto.getJobId());
+        company.setRegistrationNumber(createCompanyDto.getRegistrationNumber());
+        company.setAdress(createCompanyDto.getAddress());
+        return company;
+    }
```

Comment:
Ubiće te Jefta što u maperu praviš objekat.

Reply:
pa ovo je mapper sa createCompanyDto na Company, da sam ga imenovala createCompanyDtoToCompany ne bi imao zamerke vrv